test/js/reducers/questionnaire.spec.js

```diff
@@ -784,6 +780,7 @@ describe('questionnaire reducer', () => {
     ])
 
     const resultState = playActionsFromState(preState, reducer)([
+      actions.addLanguage('fr'),
```

Comment:
Notice that `setDefaultLanguage('fr)` does not result in adding to the list of languages AFAICT. There are a number of invariants broken each time `setDefaultLanguage` and `addLanguage` are dispatched (I'll comment below).